src/translations/en.json

```
@@ -926,6 +926,11 @@
           },
           "learn_more": "Learn more"
         },
+        "updates": {
+          "supervisor_error_title": "Could not fetch supervisor updates",
```

Supervisor is an implementation detail.
```suggestion
          "supervisor_error_title": "Unable to fetch available updates",
```

---

The idea behind that was for when we expand it, but I guess then we will be looking at hass.states anyway.